src_assets/linux/misc/postinst

```diff
@@ -1,16 +1,20 @@
 #!/bin/sh
 
 # Ensure Sunshine can grab images from KMS
-path_to_setcap=$(which setcap)
-path_to_sunshine=$(readlink -f $(which sunshine))
-if [ -x "$path_to_setcap" ] ; then
-  echo "$path_to_setcap cap_sys_admin+p $path_to_sunshine"
-        $path_to_setcap cap_sys_admin+p $path_to_sunshine
-fi
+if grep -q "OSTREE" /etc/os-release; then
+  echo "WARNING: Atomic desktop operating system detected. Skipping post-install steps. They may need to be run manually if you run into issues. See RPM install documentation for more info."
```

Comment:
```suggestion
if grep -q "OSTREE" /etc/os-release; then
  echo "WARNING: Atomic desktop operating system detected. Skipping post-install steps. They may need to be run manually if you run into issues. See RPM install documentation for more info."
  exit 0
fi
```

This will help reduce nesting.

Also, all that was recently removed from our docs since all of our packages handled it automatically... so this message is probably not adequate. Currently there is only the setcap info. https://docs.lizardbyte.dev/projects/sunshine/en/master/md_docs_2getting__started.html#linux-1

Reply:
Are you using x11?

Reply:
Yes, I'm using x11

Reply:
setcap step is for kms/wayland

Reply:
Removing the setcap step will make KMS-based capture impossible. (This is of course fine if it's technically impossible.)

Since this is the only universal capture method, I think the message should be more explicit in what is supported and what isn't.

If I'm not mistaken that leaves:
- `wlroots` for specific Wayland compositors (but not Gnome or KDE)
- `x11` (Xorg)
- NvFBC (limited to Xorg and a single vendor)

Reply:
Okay, I'll test with Wayland, and update the PR if needed

Reply:
Yeah, so I ran some tests, and as expected `setcap` cannot make changes to `/usr/bin/sunshine` since the `/usr/bin` dir is read-only.

```bash
sudo setcap cap_sys_admin+p $(readlink -f $(which sunshine))
Failed to set capabilities on file '/usr/bin/sunshine-v2024.805.184417': Read-only file system
```

*However*, when I copied the `sunshine` binary to my home dir and ran `sudo setcap cap_sys_admin+p ~/sunshine`, it worked. Running this version of `./sunshine` works perfectly on Wayland. I'm successfully able to stream with audio.

I'm not familiar with how RPMs work, but is it possible to install sunshine to a different path instead of `/usr/bin`? https://docs.fedoraproject.org/en-US/fedora-silverblue/technical-information/

One (potentially hacky) solution could be to copy the installed binary to a writable dir and then run the `setcap` command. We would also need to update the bin path in `sunshine.service` in this case.

Reply:
I don't know where you are supposed to install user packages, but I remember this discussion from a while back. https://github.com/LizardByte/Sunshine/issues/210

Reply:
Thanks, yeah the path `/usr/bin` is the right path to use from what I understand.

Installing to `/usr/bin` works for other projects. For `sunshine` it works too, it's just that we need to run `setcap` which make changes to `/usr/bin/sunshine`. It's a unique step which other packages do not need to do.